{{ cookiecutter.__repository }}/pyproject.toml

```diff
@@ -1,2 +1,21 @@
 [tool.cruft]
 skip = ["pkglist.txt", "README.md", "{{ cookiecutter.__pkg_name }}", "tests"]
+
+[tool.ruff]
+line-length = 120
+
+[tool.ruff.format]
+docstring-code-format = true
+quote-style = "double"
+indent-style = "space"
+
+[tool.ruff.lint]
+# Add the `line-too-long` rule to the enforced rule set. By default, Ruff omits rules that
+# overlap with the use of a formatter, like Black, but we can override this behavior by
+# explicitly adding the rule.
+extend-select = ["E501", "D", "DOC"]
```

Comment:
Adding `"I"` to `extend-select` would let us check import order without isort. Adding `"I"` to `fixable` enables ruff to sort the imports as well, without enabling other lint fixes.